tests/integration/pilot/revisioned_upgrade_test.go

```diff
@@ -19,18 +19,27 @@ import (
 	"fmt"
 	"strings"
 	"testing"
+	"time"
 
 	"github.com/hashicorp/go-multierror"
 
 	"istio.io/istio/pkg/config/protocol"
 	"istio.io/istio/pkg/test/framework"
 	"istio.io/istio/pkg/test/framework/components/echo"
 	"istio.io/istio/pkg/test/framework/components/echo/echoboot"
+	"istio.io/istio/pkg/test/framework/components/echo/util/traffic"
 	"istio.io/istio/pkg/test/framework/components/namespace"
 	kubetest "istio.io/istio/pkg/test/kube"
+	"istio.io/istio/pkg/test/scopes"
 	"istio.io/pkg/log"
 )
 
+const (
+	callInterval     = 200 * time.Millisecond
+	waitTimeout      = 30 * time.Second
+	successThreshold = 95
```

Comment:
Since we don't expect traffic failures during revisioned upgrade is this margin just to account for random failures and avoid flakiness?

Reply:
Yes, exactly

Reply:
Yes, exactly